components/sandbox/base/monitoring/sandbox-registration-service-proxy.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  namespace: toolchain-host-operator
+  name: sandbox-registration-service-proxy
+  labels:
+    prometheus: appstudio-workload
+spec:
+  endpoints:
+    interval: 15s
+    scheme: http
+    path: /metrics
+    port: proxy-metrics
+  namespaceSelector:
+    matchNames:
+      - toolchain-host-operator
```

Comment:
I don't think you need the namespace selector.

Reply:
done in [d293689](https://github.com/redhat-appstudio/infra-deployments/pull/2577/commits/d293689b2791c4af598a7bb70759b9505c037b11)